keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandler.kt

```diff
@@ -1009,7 +1010,15 @@ class ClusterHandler(
   }
 
   /**
-   * Translates a LaunchConfiguration object to a ClusterSpec.LaunchConfigurationSpec with default values omitted for export.
+   * Translates a Health object to a ClusterSpec.HealthSpec with default values omitted for export.
+   */
+  private fun Health.toSpecWithoutDefaults(): HealthSpec? {
+    val default = Health()
+    return buildSpecFromDiff(default, this)
+  }
```

Comment:
This was in the `Health` class previously but makes more sense as a private extension method here